website/docs/getting-started/setup/installation-guides/aws-ecs-with-fargate.md

```diff
@@ -0,0 +1,218 @@
+
+Appsmith is a web application that requires ports 80 and 443 for HTTP access. Please follow the steps [detailed here](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/working-with-security-groups.html#creating-security-group) to create a new security group.
+
+While creating the new security group, please follow the steps [detailed here](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/working-with-security-groups.html#adding-security-group-rule) to edit the "Inbound Rules" and make ports 80, 443 accessible from anywhere.
+
+### 3: Provision of an Application Load Balancer
+
+You can skip this step if you already have an ALB, but please ensure the listeners for ports 80 and 443 are available.
+
+### 4: Create an EFS filesystem
+
+1. Navigate to **AWS EFS** on the console and hit the *Create* button.
+2. Set the parameters like VPC (should be the same as the ECS cluster) and storage class as suiting your requirement.
+
+:::caution
+Please ensure your EFS, ECS cluster, and Fargate instances are all in the same VPC.
+:::
+      
+3. Click on the EFS created and navigate to the Network tab to ensure mount-target is made in the same availability zone as the ECS cluster.
+4. Create a new security group to allow inbound and outbound NFS traffic.
+5. Attach the security group to the EFS mount-target.
+
+### 5:  Create the ECS task execution role:
+
+1. Go to the **IAM** console and select **Roles**.
+2. Click Create Role.
+3. Select trust entity:
+
+    1. Select the Trusted entity type as AWS Service.
+    2. Select **Elastic Container Service Task** as the use case, and hit next.
+4. Add permission
+
+    1. Add AmazonECSTaskExecutionRolePolicy.
+    2. Add SecretsManagerReadWrite.
+    3. Click Create Policy to open the policy editor, choose JSON mode.
+
+```
+    {
+      "Version": "2012-10-17",
+      "Statement": [
+          {
+              "Effect": "Allow",
+              "Action": [
+                  "ssmmessages:CreateControlChannel",
+                  "ssmmessages:CreateDataChannel",
+                  "ssmmessages:OpenControlChannel",
+                  "ssmmessages:OpenDataChannel"
+              ],
+              "Resource": "*"
+          }
+        ]
+      }
+```
+
+    
+### 6: Provision an external MongoDB v5.0 instance
+
+Generate the MongoDB URI and ensure the credential used has the ClusterMonitor Role attached.
+
+:::caution
+   Please don't run the built-in embedded MongoDB with the EFS, this could cause the Appsmith instance to crash.
+:::
+
+## ECS overview
+
+![ECS OVERVIEW](/img/ecs-overview.png)
+
+## Deploy Appsmith (using Fargate):
+   
+1. [Create an ECS Cluster](#step-1-create-an-ecs-cluster)
+2. [Create Task and Container Definitions](#step-2-create-task-and-container-definitions)
+3. [Create and Run an ECS Service](#step-3-create-and-run-an-ecs-service)
+
+Switch to the old AWS console UI to follow the steps below.
+
+### Step 1: Create an ECS Cluster
```

Comment:
```suggestion
### 1. Create an ECS Cluster
```

Let's remove "Step" everywhere

Reply:
I have followed the same structure as in EC2. and `Step1:` looks better on the right sidebar, giving a sense of clarity to the user

Reply:
Yeah I agree but in rehaul we decided to remove the "Step" in the sub-headings as it looks necessary. Ordered list itself means users have to following them in an order. In fact we may have to completely remove those numbering from the main headings - https://developers.google.com/style/headings and just have numbers in the steps under each sub heading

Check https://docs.appsmith.com/getting-started/setup/installation-guides/kubernetes